server/routes/journeys/referral/area-of-work/tests.cy.ts

```diff
@@ -0,0 +1,41 @@
+context('Make a Referral Journey', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubGetPrisoner')
+    cy.task('stubGetPrisonerImage')
+    cy.task('stubComponents')
+    cy.task('stubAreaOfWork')
+  })
+
+  it('test area-of-work, including all edge cases', () => {
+    setupDataSignInAndGo()
+
+    checkValidation()
+
+    checkValuesPersist()
+  })
+
+  const checkValuesPersist = () => {
+    cy.findByRole('combobox', { name: /which area do you work in\?/i }).select(1)
+    cy.findByRole('button', { name: /continue/i }).click()
+    cy.findByRole('link', { name: /^back/i }).click()
+    cy.findByRole('combobox', { name: /which area do you work in\?/i }).should('have.value', 'A')
+  }
+
+  const checkValidation = () => {
+    cy.findByRole('heading', { name: /which area do you work in\?/i }).should('be.visible')
+    cy.findByRole('button', { name: /continue/i }).click()
+    cy.get('.govuk-error-summary a').should('have.length', 1)
+    cy.findByRole('link', { name: /select your area of work/i }).should('be.visible')
+    cy.contains('p', /select your area of work/i).should('be.visible')
```

Comment:
can we click on this link and check that the correct input component is focused?

Reply:
I completely forgot about that functionality - we should add this into our tests...

Reply:
Done :)